include/envoy/common/platform.h

```diff
@@ -157,7 +157,17 @@ struct msghdr {
 
 namespace Platform {
 constexpr absl::string_view null_device_path{"NUL"};
+
+constexpr bool win32SupportsOriginalDestination() {
+#if defined(WIN32) && defined(SIO_QUERY_WFP_CONNECTION_REDIRECT_RECORDS) && defined(SO_ORIGINAL_DST)
```

Comment:
This section is already guarded by MSC_VER. Do we need the WIN32 check?

Reply:
yeah left it from copy past